torch/onnx/symbolic_opset9.py

```diff
@@ -634,6 +634,7 @@ def neg(g: jit_utils.GraphContext, self):
 @_onnx_symbolic("aten::sqrt")
 @_beartype.beartype
 def sqrt(g: jit_utils.GraphContext, self):
+    # TODO(justinchuby): Handel integer inputs
```

Comment:
```suggestion
    # TODO(justinchuby): Handle integer inputs
```

Reply:
What would happen if an integer is the input?

Reply:
Since onnx sqrt only takes floats, I think we may need some casting for integers https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Sqrt-13